cli/waiter/token_post.py

```diff
@@ -199,6 +212,8 @@ def add_arguments(parser):
     format_group.add_argument('--json', help='provide the data in a JSON file', dest='json')
     format_group.add_argument('--yaml', help='provide the data in a YAML file', dest='yaml')
     format_group.add_argument('--input', help='provide the data in a JSON/YAML file', dest='input')
+    parser.add_argument('--output', help='outputs the computed token configuration in a JSON/YAML file without '
```

Comment:
Can we specify in the help message how to output to stdout by setting the output flag to `"-"`

Reply:
Updated.